### PR TITLE
Change example of vocabulary

### PR DESCRIPTION
**Pull request by peter-gribanov**
See #92

### CODE REVIEW

**Comment by peter-gribanov** [comment]
It would be great to show that these functions return the same thing, but they are created in different contexts. Like this:

**Bad:**

```php
class FirstService
{
    public function getUserInfo(): User
    {
        // ...
    }
}

class SecondService
{
    public function getUserData(): User
    {
        // ...
    }
}

class ThirdService
{
    public function getUserRecord(): User
    {
        // ...
    }
}
```

**Good:**

```php
class FirstService
{
    public function getUser(): User
    {
        // ...
    }
}

class SecondService
{
    public function getUser(): User
    {
        // ...
    }
}

class ThirdService
{
    public function getUser(): User
    {
        // ...
    }
}
```

But i'm afraid that this too greatly complicates the perception.

**Comment by Miguel-Serejo** [comment]
I think I have a different interpretation of the rule. When I read it, I thought of this:

```
Bad:
getUserInfo()
getClientData()
getCustomerRecord()

Good:
getUserInfo()
getClientInfo()
getCustomerInfo()
```

Users, clients and customers might all be different models and the rule doesn't say "turn all your different models into one model".

**Comment by peter-gribanov** [comment]
>Users, clients and customers might all be different models

Yes. This is possible, but in this example it is the same model.

Rule says:

> Use the **same** vocabulary for the **same** type of variable

**Comment by Miguel-Serejo** [comment]
I guess in my interpretation Info would be the type.


As in, User, Client and Customer are all different models with a common
attribute of type Info.

Your distinction between Services might make sense, but I feel it may be
out of scope.
When using external services, you rarely have the opportunity to determine
what the methods are named, so what the rule would compel you to do would
be something like
```
$userInfo = Service1::userInfo();
$clientInfo = Service2::clientData();
$customerInfo = Service3::customerRecord();
```
Obviously if you're developing services, you should keep the naming
consistent as you've shown, so long as that makes sense.
I'd expect if you're developing 3 services and feel the need to distinguish
between users, clients and customers, you'll have valid reasons to do so,
while it might be harder to justify using Info, Data, and Record to mean
basically the same thing (some information related to a model).

Maybe I'm just getting this whole thing backwards.

**Comment by peter-gribanov** [comment]
If i need to distinguish between users, clients and customers, this means that they are in different [Bounded contexts](https://martinfowler.com/bliki/BoundedContext.html).
I'll just create separate models - `User`, `Client`, `Customer`.

**Comment by TomasVotruba** [comment]
Nice little improvement.

Thank you :clap:
